OpenRA.Mods.Common/AI/AttackOrFleeFuzzy.cs

```diff
@@ -23,104 +23,92 @@ sealed class AttackOrFleeFuzzy
 	{
 		static readonly string[] DefaultRulesNormalOwnHealth = new[]
 		{
-			"if ((OwnHealth is Normal) " +
-			"and ((EnemyHealth is NearDead) or (EnemyHealth is Injured) or (EnemyHealth is Normal)) " +
-			"and ((RelativeAttackPower is Weak) or (RelativeAttackPower is Equal) or (RelativeAttackPower is Strong)) " +
-			"and ((RelativeSpeed is Slow) or (RelativeSpeed is Equal) or (RelativeSpeed is Fast))) " +
+			"if ((OwnHP is Normal) " +
+			"and ((EnemyHP is NearDead) or (EnemyHP is Injured) or (EnemyHP is Normal)) " +
+			"and ((RelAtkPwr is Weak) or (RelAtkPwr is Equal) or (RelAtkPwr is Strong)) " +
```

Comment:
I don't buy that these would make any measurable performance difference, and IMO the shortened versions are a lot less readable.  Can we please drop these renames?